policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/phase_in/earnings_limit.yaml

```
@@ -0,0 +1,15 @@
+description: Under the Family Security Act 2.0, the Child Tax Credit will phase-in linearly until this earnings level.
```

Also I think `phase-in` is a noun, `phase in` would be the verb
```suggestion
description: Senator Mitt Romney proposed phasing in the Child Tax Credit linearly from zero to this earnings level under the Family Security Act 2.0.
```